src/connectors/klassikradio.de.js

```diff
@@ -0,0 +1,9 @@
+'use strict';
```

Comment:
Should these changes be here?

Reply:
They should not. I have a big problem with mixed commits. I'll do a cleanup and submit again. ;)